site/examples/ExamplesPage.js

```diff
@@ -22,6 +22,7 @@ var ExampleHeader = require('./ExampleHeader');
 var ExamplesWrapper = require('./ExamplesWrapper');
 var React = require('react');
 var Constants = require('../Constants');
+const Dimensions = require('react-dimensions');
```

Comment:
Switched the examples page to use react-dimensions as well to make life easier.  

One downside is react-dimensions now spits out this warning on all the examples pages
https://github.com/digidem/react-dimensions/issues/35